### DISCUSSION TITLE
Remove Citadel mk1

### DISCUSSION THREAD

**Discussion by Kami-Interface** [Team-QUIET/QUIET-Community-Edition#131] [Ideas]
It's just a Fatboy flipped upside down... Is nothing sacred?
 Even though both Citadel mk1 and mk2 are kitbashed, at least the mk2 is done tastefully, making me think of the Covenant transport from Halo.

**Comment by PurpleMossClump** [comment]
I don't disagree that the Mk1 is an ugly, flipped-over flying Fatboy. Maybe it should go.
